src/main.scss

```diff
@@ -0,0 +1,11 @@
+@import "./styles/normalize.scss";
+@import "./styles/blocks/utils/_variables.scss";
+
+@import "./styles/blocks/header/header.scss";
+@import "./styles/blocks/header/header-nav.scss";
```

Comment:
Navigation is a separate block, it can be part of any other block, not only header.